kmath-tensors/src/commonTest/kotlin/space/kscience/kmath/tensors/core/TestDoubleTensorAlgebra.kt

```diff
@@ -0,0 +1,179 @@
+package space.kscience.kmath.tensors.core
+
+
+import space.kscience.kmath.tensors.core.algebras.DoubleTensorAlgebra
+import kotlin.test.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class TestDoubleTensorAlgebra {
```

Comment:
Test classes are usually made `internal` to prevent problems if someone imports a test source set.